frappe/utils/pdf.py

```diff
@@ -25,7 +25,7 @@ def get_pdf(html, options=None, output=None):
 
 	options.update({
 		"disable-javascript": "",
-		"disable-local-file-access": ""
+		"enable-local-file-access": None
```

Comment:
These options were added in https://github.com/frappe/frappe/pull/6836. Maybe we should use the `--allow` option to allow specific asset files/folders? 

ref: https://wkhtmltopdf.org/usage/wkhtmltopdf.txt